### PR TITLE
Do not keep a local Version variable

### PR DESCRIPTION
**Pull request by gsora**
Use cosmos-sdk/version package and Go's build tags to handle versioning
for us.
Makefile has been updated accordingly, inspired by
https://github.com/cosmos/gaia/blob/master/Makefile.

Output of `cnd version --full` after this change:

```
gsora@metropolis ~/c/commercionetwork> cnd version --long
name: commercionetwork
server_name: cnd
client_name: cndcli
version: 1.3.1-2-gec6631a
commit: ec6631a6a6a2905fc3ac5f4463a16b515ac2f383
build_tags: netgo
go: go version go1.13.3 linux/amd64
```

## Checklist
- [x] Targeted PR against correct branch
- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

______

For Admin Use:
- [x] Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- [x] Reviewers Assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]"

### CODE REVIEW

**Comment by RiccardoM** [comment]
Can we get this branch to merge into `version/1.3.2` instead of `master`?

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,38 @@
 PACKAGES_NOSIMULATION=$(shell go list ./... | grep -v '/simulation')
 BINDIR ?= $(GOPATH)/bin
+VERSION := $(shell echo $(shell git describe --tags) | sed 's/^v//')
+COMMIT := $(shell git log -1 --format='%H')
 
 export GO111MODULE = on
 
 include Makefile.ledger
 
-export GO111MODULE = on
```
Comment (RiccardoM) [inline Makefile]: I suggest keeping this in case a user has installed the project inside GOPATH

Comment (gsora) [reply Makefile]: Uhm, I only removed one of the two instances of `GO111MODULE`, having it before _and_ after `include Makefile.ledger` shouldn't make any difference.

**Comment by gsora** [comment]
> 
> 
> Can we get this branch to merge into `version/1.3.2` instead of `master`?

I changed the base branch, but it doesn't seem the cleanest solution possible. 

Maybe we could hold those changes until the next release, and create a new (cleaner) PR?

File: Makefile (hunk 1 above)
Comment (RiccardoM) [reply Makefile]: Sorry, didn't notice it was doubled

**Comment by RiccardoM** [comment]
@gsora After re-reviewing this I think you are right. You can pull from the `master`, solve the conflicts and then we can merge this into the `master` itself.

**Pull request merged**